oleesir/banking-app
# #164726489User should be able to see the account transaction history page.

#### What does this PR do?
implement transaction-history-template
#### Description of Task to be completed?
implement transaction-history-template
#### How should this be manually tested?
* Navigate to the UI folder and click on transaction_history.html in your browser
* You will see  transaction-history table
#### What are the relevant pivotal tracker stories?
[#164726489](https://www.pivotaltracker.com/story/show/164726489)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="Screenshot 2019-03-29 at 4 00 44 AM" src="https://user-images.githubusercontent.com/17679511/55207517-c9b03700-51da-11e9-8044-2b442f8702cc.png">
